src/sentry/static/sentry/app/components/gridEditable/styles.tsx

```diff
@@ -28,10 +25,42 @@ type GridEditableProps = {
   isDragging?: boolean;
 };
 
-export const GridPanel = styled(Panel)`
-  /* overflow: hidden; */
+export const Header = styled('div')`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: ${space(1)};
+`;
+export const HeaderTitle = styled('h2')`
+  margin: 0;
+
+  font-size: 20px;
+  font-weight: normal;
+  color: ${p => p.theme.gray3};
 `;
-export const GridPanelBody = styled(PanelBody)``;
+export const HeaderButton = styled('div')`
+  display: flex;
+  align-items: center;
+
+  color: ${p => p.theme.gray3};
+  cursor: pointer;
```

Comment:
Can/should this be a `<Button>`?

Reply:
Might be borderless `Button` but this component will be changed in an upcoming PR.